conftest.py

```diff
@@ -16,6 +18,14 @@
 ]
 
 
+@pytest.fixture()
+def bagsequence():
+    """Create a small bag object from a sequence for Dask Bag testing."""
+    toybag = from_sequence(list(range(10)))
+    return toybag
```

Comment:
We should probably drop this.

Reply:
I just did this in #6648 so I did the same over here :)